tutorials/plot_object_epochs.py

```diff
@@ -114,8 +117,9 @@
 
 ###############################################################################
 # Later on you can read the epochs with :func:`mne.read_epochs`. For reading
-# EEGLAB epochs files see :func:`mne.read_epochs_eeglab`.
-epochs = mne.read_epochs(epochs_fname)
+# EEGLAB epochs files see :func:`mne.read_epochs_eeglab`. We can also use
```

Comment:
I'm fine either way (but do any changes related to this in another PR in case other things crop up)